cmd/zpool/zpool.d/nvme_err

```diff
@@ -0,0 +1 @@
+smart
```

Comment:
You're going to need to update `cmd/zpool/Makefile.am` to include this.

Reply:
D'oh!  Fixed.